plugins/search-indexer/controllers/search-indexer.js

```diff
@@ -42,10 +42,27 @@ module.exports = {
 };
 
 const indexEntriesOfType = async (contentType) => {
-  const entries = await strapi.query(contentType).find();
+  let num = 0;
+  const allEntries = [];
 
-  return entries.map((entry) => {
-    updateInAlgolia(contentType, entry);
-    return entry.title || entry.name || entry.id;
-  });
+  while (true) {
+    const entries = await strapi
+      .query(contentType)
+      .find({ _limit: 100, _start: num });
```

Comment:
You could just use 
`.find({ _limit: 100, _start: allEntries.length() });`
and remove the num variable?